components/intake.py

```diff
@@ -140,10 +152,16 @@ def retract(self) -> None:
 
     def intake(self) -> None:
         self.direction = self.Direction.FORWARD
+        self.desired_injector_speed = (
+            0.0 if self.has_note() else self.inject_intake_speed
+        )
 
     def outtake(self) -> None:
         self.direction = self.Direction.BACKWARD
 
+    def inject(self) -> None:
```

Comment:
nit: It might be confusing to call a method on the intake `inject`. Maybe `feed_shooter` will be clearer for the team.